project/pyproject.toml.jinja

```diff
@@ -13,7 +13,7 @@ readme = "README.md"
 requires-python = "{{ python_package_requires_python }}"
 dependencies = [
     # keep-sorted start
-    "aeromancy @ git+ssh://git@github.com/quant-aq/aeromancy.git",
+    "aeromancy @ git+https://github.com/quant-aq/aeromancy@v0.2.0",
```

Comment:
Note to self: Release Aeromancy to include https://github.com/quant-aq/aeromancy/pull/21; then we can update this